Makefile

```diff
@@ -50,25 +46,9 @@ check: build-all static-check build/cover.html FORCE
 	@printf "\e[1;32m>> All checks successful.\e[0m\n"
 
 static-check: FORCE
-	@if ! hash staticcheck 2>/dev/null; then printf "\e[1;36m>> Installing staticcheck...\e[0m\n"; go install honnef.co/go/tools/cmd/staticcheck@latest; fi
-	@if ! hash exportloopref 2>/dev/null; then printf "\e[1;36m>> Installing exportloopref...\e[0m\n"; go install github.com/kyoh86/exportloopref/cmd/exportloopref@latest; fi
-	@if ! hash rowserrcheck 2>/dev/null; then printf "\e[1;36m>> Installing exportloopref...\e[0m\n"; go install github.com/jingyugao/rowserrcheck@latest; fi
-	@if ! hash unconvert 2>/dev/null; then printf "\e[1;36m>> Installing unparam...\e[0m\n"; go install github.com/mdempsky/unconvert@latest; fi
-	@if ! hash unparam 2>/dev/null; then printf "\e[1;36m>> Installing unparam...\e[0m\n"; go install mvdan.cc/unparam@latest; fi
-	@printf "\e[1;36m>> gofmt\e[0m\n"
-	@if s="$$(gofmt -s -d $(GO_ALLFILES) 2>/dev/null)" && test -n "$$s"; then echo "$$s"; false; fi
-	@printf "\e[1;36m>> staticcheck\e[0m\n"
-	@staticcheck -checks 'inherit,-ST1015' $(GO_ALLPKGS)
-	@printf "\e[1;36m>> exportloopref\e[0m\n"
-	@exportloopref $(GO_ALLPKGS)
-	@printf "\e[1;36m>> rowserrcheck\e[0m\n"
-	@rowserrcheck $(GO_ALLPKGS)
-	@printf "\e[1;36m>> unconvert\e[0m\n"
-	@unconvert $(GO_ALLPKGS)
-	@printf "\e[1;36m>> unparam\e[0m\n"
-	@unparam $(GO_ALLPKGS)
-	@printf "\e[1;36m>> go vet\e[0m\n"
-	@go vet $(GO_BUILDFLAGS) $(GO_ALLPKGS)
+	@if ! hash golangci-lint 2>/dev/null; then printf "\e[1;36m>> Installing golangci-lint...\e[0m\n"; curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin; fi
```

Comment:
We could also do ``go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest`` to build the go binary ourselves instead of downloading prebuilds.

Reply:
I tried that beforehand, it is really slow (~200 sec) compared to binary download (~6 sec) because `golangci-lint` has a lot of dependencies that need to be downloaded during the build process.

Reply:
For me it only took 30s to download and compile everything. Lets let Stefan decide that.

Reply:
Sure.

Reply:
I'm always wary of `curl | sh` stuff. Let's go with the `go install` variant. Also please adjust the progress display to include "(this may take a while)".